src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,29 @@
+.col
+  .row
+    .col
+      .progress{ style: "height: 8px"}
+        - requests_by_percentage.each do |state, percentage|
+          %div{ title: "#{requests[state]} #{state} requests", class: "progress-bar bg-#{request_bootstrap_color(state)}", "aria-valuemax": "100", "aria-valuemin": "0", "aria-valuenow": percentage, role: "progressbar", style: "width: #{percentage}%"}
```

Comment:
there is a bootstrap class for the width :wink:

Reply:
Only in increments of 25% though, so if it is the case for this, then perfect!

Reply:
Then it's fine like it is